armi/physics/fuelPerformance/tests/test_fuelPerformanceUtils.py

```diff
@@ -0,0 +1,36 @@
+"""
+Tests for fuel performance utilities.
+"""
+
+import unittest
+
+from armi.physics.fuelPerformance import utils
+from armi.reactor.flags import Flags
+from armi.reactor.tests import test_blocks
+
+
+class TestFuelPerformanceUtils(unittest.TestCase):
+    def test_enforceBondRemovalFraction(self):
+        b = test_blocks.loadTestBlock()
+        bond = b.getComponent(Flags.BOND)
+        bondRemovalFrac = 0.705
+        ndensBefore = b.getNumberDensity("NA")
+        bondNdensBefore = bond.getNumberDensity("NA")
+        b.p.bondBOL = bondNdensBefore
+        utils.enforceBondRemovalFraction(b, bondRemovalFrac)
+        bondNdensAfter = bond.getNumberDensity("NA")
+        ndensAfter = b.getNumberDensity("NA")
+
```

Comment:
this is just in a unit test so that's on the developers. The framework code is not sodium specific. I prefer to keep this test lean and not add too many extraneous functionality to minimize test mainteanance burden if you don't mind.

Reply:
That's fine we can leave it as is.